src/Polynomials.jl

```diff
@@ -67,7 +67,7 @@ julia> q / 2
 Poly(0.5 - 0.5⋅x^2)
 ```
 """
-struct Poly{T}
+struct Poly{T} <: Number
```

Comment:
I think this would be the first `Number` that when iterated does not just yield itself. Wonder whether that leads to any surprises...

Reply:
Yes, that is the kind of comment that makes me wonder if this request is a good idea or not. It seems harmless at first glance, but...